core/storage/feedback/gae_models.py

```diff
@@ -265,9 +265,19 @@ def _generate_id(cls, exploration_id, thread_id, message_id):
 
     @property
     def exploration_id(self):
+        """Extracts the exploration id from the message id.
```

Comment:
The caller doesn't care where the exploration ID comes from. See similar notes above here: https://github.com/oppia/oppia/pull/4841/files#diff-a667ca2415a9325db50285e4b2bea3d3R417

Ditto below. If these are property methods then you can constructs like "this thread".

Reply:
Done!